src/tespy/tools/fluid_properties.py

```diff
@@ -1729,3 +1729,35 @@ def ds_mix_pdT(flow, T):
     """
     d = 0.1
     return (s_mix_pT(flow, T + d) - s_mix_pT(flow, T - d)) / (2 * d)
+
+# %%
+
+
+def calc_physical_exergy(conn, pamb, Tamb):
+    r"""
+    Calculate specific physical exergy rounded on the 6th decimal.
+
+    Parameters
+    ----------
+    conn : tespy.connections.connection
+        Connection to calculate specific physical exergy for.
+
+    pamb : float
+        Ambient pressure pamb / Pa.
+
+    Tamb : float
+        Ambient temperature Tamb / K.
+
+    Returns
+    -------
+    ex_ph : float
+        Specific physical exergy ex_ph / (J/kg).
+
+        .. math::
+
+            ex_{ph} = (h - h(p_{amb}, T_{amb})) -
+            T_{amb} \cdot (s - s(p_{amb}, T_{amb}))
+    """
+    hamb = h_mix_pT([0, pamb, 0, conn.fluid.val], Tamb)
+    samb = s_mix_pT([0, pamb, 0, conn.fluid.val], Tamb)
+    return round((conn.h.val_SI - hamb) - Tamb*(conn.s.val_SI - samb), 6)
```

Comment:
- I am not sure, if we want to round here. I suggest to not round here, but at a later state if necessary.
- Missing spaces around `...Tamb * (...`